### PR TITLE
[3652] Add Display Format property to grouped checkbox

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/3652

### CODE REVIEW

File: static-assets/components/cstudio-forms/controls/checkbox-group.js [hunk 1]
```diff
@@ -16,468 +16,516 @@
...
+    this.value = "_not-set";
+    this.form = form;
+    this.id = id;
+    this.readonly = readonly;
+    this.minSize = 0;
+    this.hiddenEl = null;
+    // Stores the type of data the control is now working with (this value is fetched from the datasource controller)
+    this.dataType = null;
+    this.supportedPostFixes = ["_o"];
+    this.i18n = CrafterCMSNext.i18n;
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/checkbox-group.js]: don’t use i18n libs as an own prop. create closures to use it locally.

File: static-assets/components/cstudio-forms/controls/checkbox-group.js [hunk 2]
```diff
@@ -16,468 +16,516 @@
...
+  renderItem: function(item, groupEl, newValue) {
+    let rowEl, labelEl, textEl, inputEl;
+    const _self = this;
 
-                    if(_self.readonly == true){
-                        inputEl.disabled = true;
-                    }
+    rowEl = document.createElement("span");
+    rowEl.className = "checkbox";
+    // rowEl.setAttribute("for", _self.id + "-" + item.key);
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/checkbox-group.js]: x

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/components/cstudio-forms/controls/checkbox-group.js (hunk 1 above)
Comment (jvega190) [reply static-assets/components/cstudio-forms/controls/checkbox-group.js]: I created a closure for the control to use the i18n lib locally

**Comment by sumerjabri** [comment]
@rart is this ready now?

**Review by rart — APPROVED** [review]

**Pull request merged**